ur_controllers/src/scaled_joint_trajectory_controller.cpp

```diff
@@ -55,24 +55,28 @@ controller_interface::return_type ScaledJointTrajectoryController::update()
   }
 
   if (get_current_state().id() == lifecycle_msgs::msg::State::PRIMARY_STATE_INACTIVE) {
-    if (!is_halted) {
-      halt();
-      is_halted = true;
-    }
     return controller_interface::return_type::OK;
   }
 
-  auto resize_joint_trajectory_point = [](trajectory_msgs::msg::JointTrajectoryPoint& point, size_t size) {
+  auto resize_joint_trajectory_point = [&](trajectory_msgs::msg::JointTrajectoryPoint& point, size_t size) {
     point.positions.resize(size);
-    point.velocities.resize(size);
-    point.accelerations.resize(size);
+    if (has_velocity_state_interface_) {
+        point.velocities.resize(size);
+      }
+      if (has_acceleration_state_interface_) {
+        point.accelerations.resize(size);
+      }
```

Comment:
Formating broken. In general please run clang-format over your commit using the provided .clang-format file

Reply:
@kbogert An easy way to help with CI passing is to use `pre-commit`, check [this section](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver#contributor-guidelines) of the readme. If you install `pre-commit` before you change the repos file and add `#include <vector>`, it'll format it for you when you commit those changes